Examples/WriteApiExample.cs

```diff
@@ -0,0 +1,330 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using InfluxDB.Client;
+using InfluxDB.Client.Api.Domain;
+using InfluxDB.Client.Core;
+using InfluxDB.Client.Writes;
+
+namespace Examples
+{
+    public static class WriteApiExample
+    {
+        private static readonly DateTime EpochStart = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
+
+        [Measurement("influxPoint")]
+        private class InfluxPoint
+        {
+            [Column("writeType", IsTag = true)] public string WriteType { get; set; }
+
+            [Column("value")] public double Value { get; set; }
+
+            [Column(IsTimestamp = true)] public DateTime Time { get; set; }
+        }
+
+        public static async Task Main()
+        {
+            Console.WriteLine(
+                "================================= BasicEventHandler =================================");
+            await BasicEventHandler();
+            Console.WriteLine(
+                "================================ CustomEventListener ================================");
+            await CustomEventListener();
+        }
+
+        private static Task BasicEventHandler()
+        {
+            var influxDbClient = InfluxDBClientFactory.Create("http://localhost:9999",
```

Comment:
Please rename to to `client` and use the `using` keyword:

```suggestion
            using var client = InfluxDBClientFactory.Create("http://localhost:9999",
```